packages/relay/src/api/index.ts

```diff
@@ -10,10 +10,75 @@ import {
   SignedMessagePayload,
   SignTypedDataPayload,
   SignMessagePayload,
+  CreateRelayerRequest,
+  RelayerListResponse,
+  UpdateRelayerPoliciesRequest,
+  UpdateRelayerRequest,
 } from '../relayer';
 
 export const RelayerApiUrl = () => process.env.DEFENDER_RELAY_API_URL || 'https://api.defender.openzeppelin.com/';
 
+export class RelayClient extends BaseApiClient {
+  protected getPoolId(): string {
+    return process.env.DEFENDER_RELAY_CLIENT_POOL_ID || 'us-west-2_94f3puJWv';
+  }
+
+  protected getPoolClientId(): string {
+    return process.env.DEFENDER_RELAY_CLIENT_POOL_CLIENT_ID || '40e58hbc7pktmnp9i26hh5nsav';
+  }
+
+  protected getApiUrl(): string {
+    return process.env.DEFENDER_RELAY_CLIENT_API_URL || 'https://defender-api.openzeppelin.com/';
+  }
+
+  public async get(relayerId: string): Promise<RelayerModel> {
+    return this.apiCall(async (api) => {
+      return await api.get(`/relayers/${relayerId}`);
+    });
+  }
+
+  public async list(): Promise<RelayerListResponse> {
+    return this.apiCall(async (api) => {
+      return await api.get('/relayers/summary');
+    });
+  }
+
+  public async create(relayer: CreateRelayerRequest): Promise<RelayerModel> {
+    return this.apiCall(async (api) => {
+      return await api.post('/relayers', relayer);
+    });
+  }
+
+  public async update(relayerId: string, relayerUpdateParams: UpdateRelayerRequest): Promise<RelayerModel> {
+    if (relayerUpdateParams.policies) {
+      const updatedRelayer = await this.updatePolicies(
+        relayerId,
+        relayerUpdateParams.policies as UpdateRelayerPoliciesRequest,
```

Comment:
⚠️⚠️⚠️ Shouldn't be merging with past policies?

```suggestion
        { ...currentRelayer.policies, relayerUpdateParams.policies }
```

I think the casting might be sending `undefined` if any policy is not present, overriding it, and this is potentially destructive